cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/validator/AwsDataAccessRolePermissionValidator.java

```diff
@@ -0,0 +1,47 @@
+package com.sequenceiq.cloudbreak.cloud.aws.validator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.common.api.cloudstorage.StorageLocationBase;
+import com.sequenceiq.common.model.FileSystemType;
+
+@Component
+public class AwsDataAccessRolePermissionValidator extends AwsIDBrokerMappedRolePermissionValidator {
+    // The data access and ranger audit lists need to match the IDBMMS lists
+    // https://github.infra.cloudera.com/thunderhead/thunderhead/blob/master/
+    //  services/idbrokermappingmanagement/src/main/java/com/cloudera/thunderhead/
+    //  service/idbrokermappingmanagement/server/IdBrokerMappingManagementService.java#L109
+    private static final List<String> DATA_ACCESS_USERS = Arrays.asList("hbase", "hdfs", "hive",
```

Comment:
#6700 has been merged, so please free to use `AccountMappingSubject.DATA_ACCESS_USERS` instead.